doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-coverage-report-dmn-based-proc.adoc

```diff
@@ -1,7 +1,9 @@
 [id='test-scenarios-coverage-report-dmn-based-proc']
 = Generating coverage reports for DMN-based test scenarios
 
-Follow the procedure below to generate a coverage report for DMN-based test scenarios.
+In DMN-based test scenarios, the *Coverage Report* tab contains the detailed information about the number of available decisions, number of executed decisions, percentage of executed decisions, how many times each decision has been evaluated, and which decisions are evaluated for each defined test scenario in the table. The percentage of the executed decisions is also represented as a pie chart in the same *Coverage Report* tab.
```

Comment:
I would like to ask if it is intentional we use two terms "executed decisions" and "evaluated decisions". After reading the paragraph I have feeling both "executed" and "evaluated" have same meaning. What do you think about unification?

@danielezonca maybe I am missing something, what do you think?